tutorials/tmva/TMVA_SOFIE_RSofieReader.C

```diff
@@ -0,0 +1,67 @@
+/// \file
+/// \ingroup tutorial_tmva
+/// \notebook -nodraw
+/// This macro provides an example of using a trained model with Keras
+/// and make inference using SOFIE with the RSofieReader class
+/// This macro uses as input a Keras model generated with the
+/// TMVA_Higgs_Classification.C tutorial
+/// You need to run that macro before to generate the trained Keras model
+///
+///
+/// Execute in this order:
+/// ```
+/// root TMVA_Higgs_Classification.C
+/// root TMVA_SOFIE_RSofieReader.C
+/// ```
+///
+/// \macro_code
+/// \macro_output
+/// \author Lorenzo Moneta
+
+using namespace TMVA::Experimental;
+
+
+using namespace TMVA::Experimental;
+
+void TMVA_SOFIE_RSofieReader(){
+
+   RSofieReader model("Higgs_trained_model.h5");
+
+   // the input shape for this model is a tensor with shape (1,7)
+
+   std::vector<float> input = {0.1,0.2,0.3,0.4,0.5,0.6,0.7};
+
+   // predict model on a single event (takes a std::vector<float>)
+
+   auto output = model.Compute(input);
```

Comment:
This line gives an error. As inside the RSofieReader, the RModel object is named as `model`, thus probably here the model gets shadowed, thus here it should be named differently, maybe `RSofieReader reader()`.